README.md

```diff
@@ -28,4 +73,23 @@ with tracer.start_span('foo'):
 
 ## Contributing
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
\ No newline at end of file
+See [CONTRIBUTING.md](CONTRIBUTING.md)
+
+## Release Schedule
+
+OpenTelemetry Python is under active development. Our goal is to release an
+_alpha_ version of the library at the end of September 2019. This release isn't
+guaranteed to conform to a specific version of the specification, and is likely
+to include backwards-incompatible changes in later releases.
```

Comment:
```suggestion
that backwards-incompatible changes are introduced in later releases.
```

We should also mention that the alpha will only be supported until the beta comes out/not supported at all.

Reply:
That's a good thing to point out, changed in 99e6f9a.